customTyping/escodegen.d.ts

```diff
@@ -0,0 +1,145 @@
+
+declare module 'escodegen' {
```

Comment:
Can you also make a PR for this in the DefinitelyTyped repo? That way we can remove this in the future.